src/Pyz/Client/Search/Model/FactFinder/Mapper/SearchFactFinderToElasticaMapper.php

```diff
@@ -0,0 +1,173 @@
+use Spryker\Client\ProductStorage\ProductStorageClientInterface;
+
+class SearchFactFinderToElasticaMapper extends AbstractFactFinderToElasticaMapper implements FactFinderToElasticaMapperInterface
+{
+    /**
+     * @var \Elastica\ResultSet\DefaultBuilder
+     */
+    protected $elasticaDefaultBuilder;
+
+    /**
+     * @var \Spryker\Client\ProductStorage\ProductStorageClientInterface
+     */
+    protected $productStorageClient;
+
+    /**
+     * @var \Spryker\Client\ProductImageStorage\ProductImageStorageClientInterface
+     */
+    protected $productImageStorageClient;
+
+    /**
+     * @var \Spryker\Client\Locale\LocaleClientInterface
+     */
+    protected $localeClient;
+
+    /**
+     * @var string
+     */
+    protected $currentLocale;
+
+    /**
+     * @param \Elastica\ResultSet\DefaultBuilder $elasticaDefaultBuilder
+     * @param \Spryker\Client\ProductStorage\ProductStorageClientInterface $productStorageClient
+     * @param \Spryker\Client\ProductImageStorage\ProductImageStorageClientInterface $productImageStorageClient
+     * @param \Spryker\Client\PriceProductStorage\PriceProductStorageClientInterface $priceProductStorageClient
+     * @param \Spryker\Client\Locale\LocaleClientInterface $localeClient
+     */
+    public function __construct(
+        DefaultBuilder $elasticaDefaultBuilder,
+        ProductStorageClientInterface $productStorageClient,
+        ProductImageStorageClientInterface $productImageStorageClient,
+        PriceProductStorageClientInterface $priceProductStorageClient,
+        LocaleClientInterface $localeClient
+    ) {
+        parent::__construct($priceProductStorageClient);
+
+        $this->elasticaDefaultBuilder = $elasticaDefaultBuilder;
+        $this->productStorageClient = $productStorageClient;
+        $this->productImageStorageClient = $productImageStorageClient;
+        $this->localeClient = $localeClient;
+        $this->currentLocale = $this->localeClient->getCurrentLocale();
+    }
+
+    /**
+     * @param array $searchResult
+     * @param \Elastica\Query $elasticaQuery
+     *
+     * @return \Elastica\ResultSet
+     */
+    public function map(array $searchResult, Query $elasticaQuery): ResultSet
+    {
+        $elasticaResponseArray = $this->mapSearchResultToElasticaResponseArray($searchResult);
+        $elasticaResponse = new \Elastica\Response($elasticaResponseArray,200);
+
+        return $this->elasticaDefaultBuilder->buildResultSet($elasticaResponse, $elasticaQuery);
+    }
+
+    /**
+     * @param array $searchResult
+     *
+     * @return array
+     */
+    protected function mapSearchResultToElasticaResponseArray(array $searchResult): array
+    {
+
+        $elasticaResponseArray = [
+            'took' => 1,
+            'timed_out' => false,
+            '_shards' =>
+                [
+                    'total' => 1,
+                    'successful' => 1,
+                    'skipped' => 0,
+                    'failed' => 0,
+                ],
```

Comment:
Are these data used somewhere? I wouldn't provide them at all as they are incorrect and that's even more confusing than not providing them.